en/news/_posts/2017-12-25-ruby-2-5-0-released.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: news_post
+title: "Ruby 2.5.0 Released"
+author: "naruse"
+translator:
+date: 2017-12-25 00:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.5.0.
+
+Ruby 2.5.0 is the first stable release of Ruby 2.5 series.
+It introduces many new features and performance improvements, for example:
+
+## New Features
+
+* Print backtrace and error message in reverse order if STDERR is unchanged and a tty.
+  [Feature #8661] [experimental]
+
+* Top-level constant look-up is removed.  [Feature #11547]
+
+* rescue/else/ensure are allowed inside do/end blocks.  [Feature #12906]
+
+* Add yield\_self.  [Feature #6721]
+
+## Performance improvements
+
+* Dynamic instrumentation for TracePoint hooks instead of using "trace"
+  instruction to avoid overhead. [Feature #14104]
+
+* Performance of block passing using block parameters is improved by
+  lazy Proc allocation. [Feature #14045]
+
+* Mutex is rewritten to be smaller and faster. [Feature #13517]
+
+* SecureRandom now prefers OS-provided sources over OpenSSL. [Bug #9569]
+
+## Other notable changes since 2.4
+
+* Update to Onigmo 6.1.3.
+  It adds the [absence operator](https://github.com/k-takata/Onigmo/issues/87).
+  Note that Ruby 2.4.1 also includes this change.
+* Update to RubyGems 2.7.0.
+* Update to RDoc 6.0.0.
+  * Change lexer from IRB based one to Ripper;
+    this much improves the speed of generating documentation.
+    [https://github.com/ruby/rdoc/pull/512]
+  * Fix so many bugs from the last dozen years or so.
+  * Support new Ruby syntaxes from the last several years.
+* Update supported Unicode version to 10.0.0.
+
```

Comment:
Plz add `* Support branch coverage and method coverage measurement. [Feature #13901]`

Reply:
While I forgot to write NEWS, similar to RDoc's one, I would like to add a following note:

```
* ERB's rendering becomes 2x faster
```

Reply:
While I forgot to write NEWS, similar to RDoc's one, I would like to add a following note:

```
* ERB's rendering becomes 2x faster
```

Reply:
@k0kubun it's notable perfomance gain.  should be written in Perfomance implovements section.

Reply:
Thank you, moved my suggestion to https://github.com/ruby/www.ruby-lang.org/pull/1706/files#r158605774